src/app/components/Footer/index.jsx

```diff
@@ -1,27 +1,119 @@
 import React from 'react';
 import styled from 'styled-components';
 import {
-  C_STORM,
+  C_GREY,
   C_WHITE,
   FF_NEWS_SANS_REG,
+  GEL_SPACING,
   GEL_SPACING_DBL,
 } from '../../lib/constants/styles';
 
 const StyledFooter = styled.footer`
-  background-color: ${C_STORM};
+  background-color: ${C_GREY};
   font-family: ${FF_NEWS_SANS_REG};
-  padding: ${GEL_SPACING_DBL}px;
+  padding: ${GEL_SPACING_DBL}px ${GEL_SPACING}px;
+`;
+
+const StyledList = styled.ul`
+  border-bottom: 1px solid ${C_WHITE};
+  display: grid;
+  grid-auto-flow: column;
+  grid-template-columns: repeat(2, 50%);
+  grid-template-rows: repeat(4, 50%);
+  list-style-type: none;
+  margin: 0 0 ${GEL_SPACING}px 0;
+  padding: 0 0 ${GEL_SPACING}px 0;
```

Comment:
Needed to set the padding and margin 0 on top, left and right, since we are not using a reset CSS file (https://github.com/bbc/simorgh/issues/248). If we do choose to add a CSS reset file, we can remove these attributes to prevent duplication.